### PR TITLE
document extensibility of -groups parameter via providers

### PR DESCRIPTION
**Pull request by baentsch**
- Fix a typo for TLS-SIGALG
- Document extensibility of -groups parameter via TLS-GROUP

### CODE REVIEW

**Review by beldmit — DISMISSED** [review]
LGTM

**Review by paulidale — APPROVED** [review]

File: doc/man3/SSL_CONF_cmd.pod [hunk 1]
```diff
@@ -161,9 +161,10 @@ where applicable (e.g. B<X25519>, B<ffdhe2048>) or an OpenSSL OID name
 (e.g. B<prime256v1>). Group names are case sensitive. The list should be
 in order of preference with the most preferred group first.
 
-Currently supported groups for B<TLSv1.3> are B<P-256>, B<P-384>, B<P-521>,
-B<X25519>, B<X448>, B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>, B<ffdhe6144>,
-B<ffdhe8192>.
+Groups for B<TLSv1.3> in the default provider are B<P-256>, B<P-384>,
+B<P-521>, B<X25519>, B<X448>, B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>,
+B<ffdhe6144>, B<ffdhe8192>. Additional providers may make available
```
Comment (t8m) [inline doc/man3/SSL_CONF_cmd.pod:166]: We also support brainpoolP256r1tls13, brainpoolP384r1tls13 and  brainpoolP512r1tls13.

**Review by t8m — CHANGES_REQUESTED** [review]
Could you please add the brainpool groups?

**Comment by baentsch** [comment]
> Could you please add the brainpool groups?

So done.

File: doc/man3/SSL_CONF_cmd.pod (hunk 1 above)
Comment (baentsch) [reply doc/man3/SSL_CONF_cmd.pod]: Added them in latest commit. OK to omit the non-"...tls13" variants, though?

Comment (t8m) [reply doc/man3/SSL_CONF_cmd.pod]: Yes

**Review by t8m — APPROVED** [review]

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by t8m** [comment]
Squashed and merged to the master branch. Thank you.

**Pull request closed without merging**